fieldservice_portal/static/src/js/fsm_order_portal.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
```

Comment:
**issue**: pre-commit is whyning because the file bust be renamed `fsm_order_portal.esm.js`

Reply:
Updated extension with "esm.js". Thank you @PicchiSeba